packages/cardano-graphql/src/Schema/types/Transaction/TransactionInput.ts

```diff
@@ -0,0 +1,23 @@
+import { Cardano } from '@cardano-sdk/core';
+import { Field, Int, ObjectType } from 'type-graphql';
+import { Redeemer } from './Redeemer';
+import { Transaction } from './Transaction';
+import { Value } from './Value';
+
+@ObjectType()
+export class TransactionInput {
+  @Field(() => String)
+  address: Cardano.Address;
+  @Field(() => Redeemer, { nullable: true })
+  redeemer?: Redeemer;
+  // Review: what is the difference between 'transaction' and 'sourceTransaction'?
```

Comment:
`sourceTransaction` is where the output was created that now becomes the input of the `transaction`